boards/PDM/Inc/stm32f3xx_hal_conf.h

```diff
@@ -1,7 +1,7 @@
 /**
   ******************************************************************************
   * @file    stm32f3xx_hal_conf.h
-  * @brief   HAL configuration file.  
+  * @brief   HAL configuration file.
```

Comment:
Whoops. Looks like this wasn't just a formatting thing like we discussed. We should fix this.